### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_Publish Projects_ScreenReader: Voice Over announces control type of database Combo box as Button.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4891]
--- Actual:
Voice Over announces control type of database Combo box as Button.

--- Expected:
Voice Over should announce the control type for Database combo box as combo box.

**GitHubTags**: -A11yMAS;-A11yTCS;-A11ySev2;-SH_AzureDataStudioSQL_Feb2021;-SQL Azure Data Studio;-MAC;-FTP;-DesktopApp;-WCAG4.1.2;-VoiceOver;-Element:Combobox:-AINotInScope;

--- Environment Details:
Application Name: Azure Data Studio
Version: 1.26.1
Mac OS Version: Big Sur Version 11.2.2
Screen reader: Voice Over

--- Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an right click on any created project.
5. Tab till "Publish Project" and hit enter
6. Start Voice Over and tab to database combo box and check the result.

--- User Impact:
Screen reader user will get confused if Voice Over announces control type of database Combo box as Button.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
